frontend/src/lib/i18n/index.js

```diff
@@ -1,15 +1,21 @@
-import { browser } from '$app/environment';
+import {} from '$app/environment';
 import { init, register } from 'svelte-i18n';
+import { browser } from '$app/environment';
 
 const defaultLocale = 'en';
```

Comment:
Should we declare additional constant, instead of putting "en" in line 19 like: `fallbackLocale: "en"`?

Reply:
> Should we declare additional constant, instead of putting "en" in line 19 like: `fallbackLocale: "en"`?

I dont think so

Reply:
Why not? Could you explain in more detail?